.azure-pipelines/docker.yml

```diff
@@ -0,0 +1,63 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the ODPi Egeria project.
+---
+trigger:
+  paths:
+    include:
+      - open-metadata-resources/open-metadata-deployment/docker
```

Comment:
so I presume we could break down the pipeline and trigger each independently? The egeria one will change the most, the rest are less frequent? This could be a future step? - doesn't inhibit making this first step

Reply:
We could - The builds could also be configured to depend on the others being created first. Ex: publish of base container triggers building of 3 dependent containers, etc.

Reply:
ok I'll close this part of the thread, and will open a new issue if/when we want to explore improving docker build once we have the basics in place